DEVELOPER_GUIDE.md

```diff
@@ -25,6 +25,7 @@ So you want to contribute code to this project? Excellent! We're glad you're her
     - `cdk deploy OpenSearchMetrics-GitHubAutomationApp-Secret`: Creates the GitHub app secret which will be used during the GitHub app runtime.
     - `cdk deploy OpenSearchMetrics-GitHubWorkflowMonitor-Alarms`: Creates the Alarms to Monitor the Critical GitHub CI workflows by the GitHub Automation App.
     - `cdk deploy OpenSearchMetrics-GitHubAutomationApp`: Create the resources which launches the [GitHub Automation App](https://github.com/opensearch-project/automation-app). Listens to GitHub events and index the data to Metrics cluster.
+    - `cdk deploy OpenSearchMetrics-S3`: Create the S3 Bucket that will store GitHub Events.
```

Comment:
Can you please change this to `OpenSearchMetrics-GitHubAutomationAppEvents-S3`

Reply:
I'm thinking we can put various s3 buckets in this stack in the future, so the name should be kept general. I'll change the description to describe this

Reply:
Yes since you added under `infrastructure/lib/stacks/s3.ts` it will create new stack for new s3 bucket, so the stack name is `OpenSearchMetrics-GitHubAutomationAppEvents-S3`.  With this we have have a proper naming convention for each s3 bucket stack.
So tomorrow if you need a new s3 bucket it will be a new stack altogether.

Reply:
Got it, I updated the PR to make these changes